GoogleForCreators/web-stories-wp
# Template Updates: Travel

## Context

Part of 1.7 release is updating existing templates to avoid custom animations and unnecessary prepublish checklist suggestions. 

## Summary

Travel Template revisions: 
- remove custom animations and replace them with effects available in editor
- update svgs to stickers (socials and map)
- swap square colors to be black, some text color updates
- add assistive text to images.

## Relevant Technical Choices

When Ivo and I were revising the custom animations neither of us remembered that fade in is only available on foreground elements, so I chose to either zoom in or pan in for relevant background images:
- Elephant bg image will zoom in on page 1 (was supposed to fade in)
- Photographer bg will pan in from left on page 5 (was supposed to fly in)

## To-do


## User-facing changes

Travel template has been updated. 

**Before**
https://user-images.githubusercontent.com/10720454/114075533-7c034600-985a-11eb-81c6-28cca2317580.mp4

**After**
https://user-images.githubusercontent.com/10720454/114075559-845b8100-985a-11eb-899c-b15db3dadee0.mp4


## Testing Instructions

Use the travel template from dashboard/explore templates and see that template has been updated in line with Figma: https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories-Post-Stable?node-id=7357%3A546039 and as specified here: https://github.com/google/web-stories-wp/issues/6975 
There should be no more custom animations, all images should have assistive text.

### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?

No

### Does this PR change what data or activity we track or use?

No

### Does this PR have a legal-related impact?

No

## Checklist




---



Fixes #6975
